src/steamship/utils/file_tags.py

```diff
@@ -0,0 +1,14 @@
+from steamship import File, Steamship, SteamshipError, Tag
+
+
+def update_file_status(client: Steamship, file: File, status: str) -> None:
+    file = file.refresh()
+    status_tags = [tag for tag in file.tags if tag.kind == "status"]
+    for status_tag in status_tags:
+        try:
+            status_tag.client = client
+            status_tag.delete()
+        except SteamshipError:
+            pass
+
+    Tag.create(client, file_id=file.id, kind="status", name=status)
```

Comment:
Yeah I agree. This code just ports what `Ask My *` does.. but it leaves me wondering if we want some sort of operation atop tag that replaces if exist, and adds otherwise.

That still doesn't solve the ordering problem you raise here, but it would be a step better than we've got now